templates/design.md

```diff
@@ -0,0 +1,94 @@
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: (leave this empty, to be filled in later)
+- Authors: (the names of everyone contributing to this RFC)
+
+# (RFC title goes here)
+
+## Summary
+
+<!-- One paragraph explanation of the feature. -->
```

Comment:
Nit: I like "One-paragraph explanation" a little better.

Or, as a full sentence, something like: "Write a one-paragraph explanation of the feature" or "Write one paragraph explaining the feature".